### ISSUE TITLE
Sign In from Electron App

### ISSUE THREAD

**Issue by MerrickKing** [RogueException/Discord.OAuth2#13]
I've got the sign-in working fine on a normal MVC site, however if I try to sign in from an Electron then the Discord OAuth2 page thinks it's a broken Discord client (see attached). Not sure if there's anything that can be done about that from this but any ideas for getting around this would be great thanks

**Comment by MerrickKing** [comment]
![Screenshot_2](https://user-images.githubusercontent.com/11535986/61668677-5e342d00-acd5-11e9-8c5b-64fe4bb47408.png)
![Screenshot_3](https://user-images.githubusercontent.com/11535986/61668686-655b3b00-acd5-11e9-964d-8142bc340b87.png)

**Comment by SandwichDev** [comment]
@MerrickKing Did you ever find any solution for this? I'm currently having the same problems.

**Comment by MerrickKing** [comment]
@SandwichDev Nope, I've tried setting the User-Agent in both the HTML and the Electron startup code. I think that Discord is detecting that it's electron through some other method. Might see if I can open it in Chrome and get it to work like that.
